### PR TITLE
pr4

### PR DESCRIPTION
**Pull request by b454545**
why

### CODE REVIEW

**Comment by b454545** [comment]
help

**Review by ibrahimaljalal — APPROVED** [review]

File: test2.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-this is line 1 in 2
-this is line 2 in 2
-this is line 3 in 2
\ No newline at end of file
+this is line 1 in 2 bb3
+this is line 2 in 2 bb4
+this is line 3 in 2 bb3
```
Comment (ibrahimaljalal) [inline test2.txt:3]:
```suggestion
this is line 1 in 2 bb3

```

how aboyt that

File: test3.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-this is line 1 in 3
-this is line 2 in 3
-this is line 3 in 3
\ No newline at end of file
+this is line 1 in 3 b5
+this is line 2 in 3 b5
+this is line 3 in 3 bb4
```
Comment (ibrahimaljalal) [inline test3.txt:3]:
```suggestion
this is liney 1 in 3 b5r

this is linye 3 in 3 bb4 y
```

File: test2.txt (hunk 1 above)
Comment (ibrahimaljalal) [inline test2.txt:3]:
```suggestion
ibrahims change
```

File: test1.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
-this is line 1 in 1
-this is line 2 in 1
-this is line 3 in 1 change from i
\ No newline at end of file
+this is line 1 in 1 bb
+this is line 2 in 1 bb2
```
Comment (ibrahimaljalal) [inline test1.txt:2]: test

File: test1.txt [hunk 4]
```diff
@@ -1,3 +1,3 @@
-this is line 1 in 1
-this is line 2 in 1
-this is line 3 in 1 change from i
\ No newline at end of file
+this is line 1 in 1 bb
```
Comment (ibrahimaljalal) [inline test1.txt:1]: test

**Review by ibrahimaljalal — CHANGES_REQUESTED** [review]
I dont like it

**Pull request merged**